### ISSUE TITLE
Menu slow to open when reading

### ISSUE THREAD

**Issue by neabb** [baskerville/plato#356]
With a book open, when I tap the middle of the screen, it takes approximately 10 seconds for the UI to appear.
There is some general unresponsiveness with the UI as well, but only whilst books are open.
I am running the latest version of Plato (Plato 0.9.41).

**Comment by neabb** [comment]
Update:
I obtained another version of the ebook and it runs fine.

I ran an epub check on the original and had these errors:

Validating using EPUB version 2.0.1 rules.
ERROR(RSC-005): Super awesome book.epub/OEBPS/content.opf(64,106): Error while parsing file: attribute "properties" not allowed here; expected attribute "fallback", "fallback-style", "required-modules" or "required-namespace"
ERROR(RSC-005): Super awesome book.epub/OEBPS/content.opf(75,13): Error while parsing file: element "tours" incomplete; missing required element "tour"
ERROR(RSC-005): Super awesome book.epub/OEBPS/Text/part0000.xhtml(2,72): Error while parsing file: attribute "style" not allowed here; expected attribute "dir", "lang", "version" or "xml:lang"
ERROR(RSC-005): Super awesome book.epub/OEBPS/Text/part0000.xhtml(15,97): Error while parsing file: Duplicate "kobo.5.1"
ERROR(RSC-005): Super awesome book.epub/OEBPS/Text/part0000.xhtml(18,114): Error while parsing file: Duplicate "kobo.5.1"
ERROR(RSC-007): Super awesome book.epub/OEBPS/Styles/style0001.css(5,2): Referenced resource "OEBPS/Styles/XXXXXXXXXXXXXXXX" could not be found in the EPUB.
ERROR(RSC-007): Super awesome book.epub/OEBPS/Styles/style0001.css(11,2): Referenced resource "OEBPS/Styles/XXXXXXXXXXXXXXXX" could not be found in the EPUB.


I am not sure how they could make things slow...
